v0.7/speech_recognition/rnnt/pytorch/inference.py

```diff
@@ -193,15 +230,18 @@ def main(args):
     if args.cuda:
         model.cuda()
 
-    # Ideally, I would jit this as well... But this is just the constructor...
-    greedy_decoder = RNNTGreedyDecoder(len(ctc_vocab) - 1, model)
-
+    # greedy_decoder = RNNTGreedyDecoder(len(rnnt_vocab) - 1, model)
+    model.eval()
+    # torch.nn.LSTM.__constants__.append('training')
+    greedy_decoder = ScriptGreedyDecoder(len(rnnt_vocab) - 1, torch.jit.script(model))
```

Comment:
@galv Just curious, is this functionally equivalent to the original `RNNTGreedyDecoder`? Or in other words, do they generate the same outputs give the same input?